lib/features/personal_summary/overview/widgets/monthly_bar_chart.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:flutter/material.dart';
+import 'package:givt_app/utils/utils.dart';
+import 'package:intl/intl.dart';
+
+class MonthlyBarChart extends StatelessWidget {
+  const MonthlyBarChart({
+    required this.month,
+    required this.amount,
+    super.key,
+  });
+
+  final String month;
```

Comment:
This month variable is not only the month if I read this code correctly. 

Could you change the variable name? Or maybe add some doc comment so that its clear what the format of this variable is/should be

Reply:
Yes, i should have changed it. It's done